lib/simplekiq/orchestration_executor.rb

```diff
@@ -4,26 +4,32 @@ module Simplekiq
   class OrchestrationExecutor
     def self.execute(args:, job:, workflow:)
       orchestration_batch = Sidekiq::Batch.new
-      orchestration_batch.description = "#{job.class.name} Simplekiq orchestration"
+      orchestration_batch.description = "[Simplekiq] #{job.class.name}. Params: #{Simplekiq.format_args(args)}"
       Simplekiq.auto_define_callbacks(orchestration_batch, args: args, job: job)
 
       orchestration_batch.jobs do
-        new.run_step(workflow, 0) unless workflow.empty?
+        new.run_step(workflow, 0, job.class.name, orchestration_batch.bid) unless workflow.empty?
       end
     end
 
-    def run_step(workflow, step)
+    def run_step(workflow, step, orchestration_job_class_name, orchestration_batch_bid)
       *jobs = workflow.at(step)
       # This will never be empty because Orchestration#serialized_workflow skips inserting
       # a new step for in_parallel if there were no inner jobs specified.
 
       next_step = step + 1
       step_batch = Sidekiq::Batch.new
-      step_batch.description = "Simplekiq orchestrated step #{next_step}"
+      step_batch.description = "[Simplekiq] step #{next_step} in #{orchestration_job_class_name}. " \
+        "Orchestration batch ID: #{orchestration_batch_bid}."
```

Comment:
Pretty sure sidekiq tracks parent batch IDs already (it has to for child/parent batch callbacks to fire properly)... I bet we could just pull the parent bid into the Sidekiq UI as a first-class batch property instead of shoving it into the description?

Reply:
ah true, I'll do that in the sidekiq UI and remove it from here

Reply:
(frankly kind of surprised that the sidekiq batch UI doesn't expose parent batches already, that seems like a feature we should try and get upstream)

Reply:
yeah I just saw it too. It's not on the batches table but it's there when you click on a specific batch